### ISSUE TITLE
[Bug]: Accommodation Contact Information not grayed out

### ISSUE THREAD

**Issue by MatthewGWilson** [UofT-Frosh-Orientation/orientation-website#294]
### Describe the bug

On the health and safety tab of registration the Accommodation Contact Information question title is not grayed out even when it is not selectable

### How To Reproduce

1. Go to the registration page and the Health and Safety tab
2. Scroll to the Accommodation Contact Information question and you can see that the title is not grayed out like it is on other questions that are not selectable unless you give certain previous answers

### Expected behavior

I expected the title to be grayed out like other question titles that are not selectable

### What device were you using when you encountered the error?

Dell XPS 8940

### What browser were you using when you encountered the error?

Chrome

### What operating system were you using?

Windows 10

### Additional Information

I think its probably because this is the only drop down that requires certain previous answers to be opened

**Comment by MatthewGWilson** [comment]
Shoot sorry just noticed the same problem is present on the next tab for the What is you main method of commuting to campus? question

**Comment by jameskokoska** [comment]
TLDR
Titles not greyed out when disabled for dropdowns

**Comment by jameskokoska** [comment]
https://github.com/UofT-Frosh-Orientation/orientation-website/commit/b42337234527bc8be20f9137e49cd7a83409bef4
